README.md

```diff
@@ -9,7 +9,7 @@
 </p>
 
 # About
-Stratus is a Web Platform that aims to connect organizations directly with developers to find solutions and make real world impact. It was built in response to the year 2020... what a year. This year has revealed the most vurnable out of all of us, and has left the rest of us with much stronger will to HELP, and MAKE AN IMPACT. With the world shifting online, developers are becoming a nessecity in any organization. However, it is often the organizations with the least reasources, that are in need of developers. Stratus, was created to directly link these organizations with up and coming developers, who are in need of experience, and who's skill's could level up any organization.  
+Stratus is a Web Platform that aims to connect organizations directly with developers to find solutions and make real world impact. It was built in response to the year 2020... what a year. This year has revealed the most vurnable out of all of us, and has left the rest of us with much stronger will to HELP, and MAKE AN IMPACT. With the world shifting online, developers are becoming a nessecity in any organization. However, it is often the organizations with the least reasources, that are in need of developers. Stratus, was created to directly link these organizations with up and coming developers, who are in need of experience, and who's skill's could level up any organization.
```

Comment:
What changed here?

Reply:
Nothing, I would revert the changes but I don't know how...do you know?